pages/404.tsx

```diff
@@ -46,7 +46,7 @@ const PageNotFound: NextPage = () => {
   const intl = useIntl()
 
   return (
-    <Page documentTitle={intl.formatMessage({ id: "404.documentTitle" })}>
+    <Page documentTitle={intl.formatMessage({ id: "404.documentTitle" })} useImprovedSearch>
```

Comment:
doesn't this set `useImprovedSearch` to `true` instead of passing the value of the prop?

Reply:
Yup, should be false